generative_ai/requirements.txt

```
@@ -7,6 +7,6 @@ google-cloud-aiplatform[pipelines,rapid_evaluation,reasoningengine]==1.57.0
 google-auth==2.29.0
 anthropic[vertex]==0.28.0
 langchain-core==0.1.52
```

Due to dependency min change, need to jump langchain-core. Choosing latest available. 
```suggestion
langchain-core==0.2.10
```